### DISCUSSION TITLE
Failed to get Db2 level: NOT AUTHORIZED TO COMMAND WRKPTFGRP. (42501)

### DISCUSSION THREAD

**Discussion by cwages** [codefori/vscode-ibmi#2208] [General]
we have 3 developers that are also getting this error. Failed to get Db2 level: NOT AUTHORIZED TO COMMAND WRKPTFGRP. (42501) with code4i 2.12.1 and db2i v 1.1.1. We have many others that do not get this error.

**Comment by worksofliam** [comment]
Yes, it's a known issue. Some users have authority to a certain feature and some don't. Not a pushing issue right now, unless you are finding it impacting your work?

**Reply by cwages** [reply]
one person reported a work around was to have a person not getting the issue, log in on the vs code instance with their profile. After that was done and changed back to the problem profile. It was reported as working. I haven't had a chance to verify though.
